File: submissions/example-santa/tree.kt
fun main() {
    val height = 5
    for (i in 0 until height) {
        println(" ".repeat(height - i - 1) + "*".repeat(2 * i + 1))
    }
    println(" ".repeat(height - 1) + "|")
    println("\nMerry Christmas! 🎄")
}

File: submissions/example-santa/tree.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	height := 5
	for i := 0; i < height; i++ {
		spaces := strings.Repeat(" ", height-i-1)
		stars := strings.Repeat("*", 2*i+1)
		fmt.Println(spaces + stars)
	}
	trunkSpaces := strings.Repeat(" ", height-1)
	fmt.Println(trunkSpaces + "|")
	fmt.Println("\nMerry Christmas! 🎄")
}
